src/main/resources/templates/mdBenefitsFlow/expeditedSnapMoneyOnHandAmount.html

```diff
@@ -10,12 +10,13 @@
       <main id="content" role="main" class="form-card spacing-above-35">
         <th:block th:replace="~{fragments/icons :: assetsCashAccounts}"></th:block>
         <th:block
-            th:replace="~{fragments/cardHeader :: cardHeader(header=#{expedited-money-on-hand-amount.header}, subtext=#{expedited-money-on-hand-amount.subtext})}"/>
+            th:replace="~{fragments/cardHeader :: cardHeader(header=#{expedited-money-on-hand-amount.header}, subtext=#{expedited-money-on-hand-amount.subtext}, required=true)}"/>
         <th:block
             th:replace="~{fragments/form :: form(action=${formAction}, content=~{::formContent})}">
           <th:block th:ref="formContent">
             <div class="form-card__content">
-
+              <th:block
+                  th:replace="'fragments/inputs/money' :: money(inputName='expeditedMoneyOnHandAmount', ariaLabel='header header-help-message', placeholder='0.00')"/>
```

Comment:
Weird, I'm guessing the rebase didn't go well or rather that it went fine, but now we have two money spots on this page.  I'll fix it.